arangod/Transaction/Manager.cpp

```diff
@@ -293,9 +293,9 @@ arangodb::cluster::CallbackGuard Manager::buildCallbackGuard(
       auto& clusterInfo = clusterFeature.clusterInfo();
       rGuard = clusterInfo.rebootTracker().callMeOnChange(
           origin,
-          [this, tid = state.id()]() {
+          [this, tid = state.id(), vocbase = state.vocbase().name()]() {
```

Comment:
should we better take a variable name like `databaseName` so its more clear? `vocbase` usually refers to the full db-object, right?